test/commons/utils/index.js

```diff
@@ -1,21 +0,0 @@
-describe('utils.escapeSelector', function() {
-	'use strict';
-
-	it('should be a function', function() {
-		assert.isFunction(axe.commons.utils.escapeSelector);
-	});
-});
-describe('utils.matchesSelector', function() {
-	'use strict';
-
-	it('should be a function', function() {
-		assert.isFunction(axe.commons.utils.matchesSelector);
-	});
-});
-describe('utils.clone', function() {
-	'use strict';
-
-	it('should be a function', function() {
-		assert.isFunction(axe.commons.utils.clone);
-	});
-});
```

Comment:
Same here, we don't want to make a breaking change. Just check that `assert.equal(axe.utils, axe.commons.utils)`.